File: Editor/src/main/java/org/turbomedia/turboedit/editor/windows/RenderQueueWindow.java
package org.turbomedia.turboedit.editor.windows;

import javafx.scene.Scene;
import javafx.stage.Stage;
import org.turbomedia.turboedit.editor.panes.render_queue.RenderQueuePane;

import static org.turbomedia.turboedit.editor.Editor.ICON;
import static org.turbomedia.turboedit.editor.Editor.TITLE;
import static org.turbomedia.turboedit.editor.misc.Locale.GetText;

public class RenderQueueWindow extends Stage {

    public RenderQueueWindow() {
        setTitle(TITLE + " - " + GetText("title.render_queue"));
        getIcons().add(ICON);
        setAlwaysOnTop(true);

        setWidth(500);
        setHeight(250);

        centerOnScreen();
        setScene(new Scene(new RenderQueuePane()));

        showAndWait();
    }

}

File: Editor/src/main/kotlin/turboedit/editor/windows/RenderQueueWindow.kt
package turboedit.editor.windows

import javafx.scene.Scene
import javafx.stage.Stage
import turboedit.editor.Editor
import turboedit.editor.misc.Locale
import turboedit.editor.panes.render_queue.RenderQueuePane

class RenderQueueWindow : Stage() {
    init {
        title = Editor.Companion.TITLE + " - " + Locale.getText("title.render_queue")
        icons.add(Editor.Companion.ICON)
        width = 500.0
        height = 250.0
        scene = Scene(RenderQueuePane())
        isAlwaysOnTop = true

        centerOnScreen()
        showAndWait()
    }
}
